lttoolbox/fst_processor.cc

```diff
@@ -243,6 +243,7 @@ FSTProcessor::readAnalysis(InputFile& input)
   {
     UChar32 val = input_buffer.next();
     while ((useIgnoredChars || useDefaultIgnoredChars) && ignored_chars.find(val) != ignored_chars.end()) {
+      input_buffer.back(1);
```

Comment:
wouldn't this make it so that if an ignored char ever got into `input_buffer` that we'd loop on it endlessly?

Reply:
haha thanks, that was silly. (The real fix is the one below which never adds ignored chars into input buffer, making this line both unnecessary and potentially dangerous =D)